src/components/signIn.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'
+import Button from 'react-bootstrap/Button'
+
+export default class Login extends Component {
+  render () {
+    return (
+      <form>
```

Comment:
You'll need to look for a library to use forms more easily. It's a complicated to do by hand in React. I recommend using [Formik](https://jaredpalmer.com/formik).